infra/cifuzz/actions/entrypoint.py

```diff
@@ -12,46 +12,57 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 """Builds and runs specific OSS-Fuzz project's fuzzers for CI tools."""
-
 import os
 import subprocess
 import sys
 
 
 def main():
-  """Runs OSS-Fuzz project's fuzzers for CI tools."""
-  project_name = os.environ['OSS_FUZZ_PROJECT_NAME']
-  repo_name = os.environ['GITHUB_REPOSITORY'].rsplit('/', 1)[-1]
+  """Runs OSS-Fuzz project's fuzzers for CI tools.
+
+  Required environment variables:
+    PROJECT_NAME: The name of OSS-Fuzz project.
+    FUZZ_TIME: The length of time in seconds that fuzzers are to be run.
+    GITHUB_REPOSITORY: The name of the Github repo that called this script.
+    GITHUB_SHA: The commit SHA that triggered this script.
+
+  Returns:
+    0 on success or 1 on Failure.
+  """
+  oss_fuzz_project_name = os.environ['PROJECT_NAME']
+  fuzz_time = os.environ['FUZZ_TIME']
+  github_repo_name = os.environ['GITHUB_REPOSITORY'].rsplit('/', 1)[-1]
   commit_sha = os.environ['GITHUB_SHA']
 
   # Build the specified project's fuzzers from the current repo state.
   print('Building fuzzers\nproject: {0}\nrepo name: {1}\ncommit: {2}'.format(
-      project_name, repo_name, commit_sha))
+      oss_fuzz_project_name, github_repo_name, commit_sha))
   command = [
-      'python3', '/src/oss-fuzz/infra/cifuzz.py', 'build_fuzzers', project_name,
-      repo_name, commit_sha
+      'python3', '/src/oss-fuzz/infra/cifuzz.py', 'build_fuzzers',
```

Comment:
Why is this file is calling cifuzz.py through subprocess instead of using it as a module.
Also, it might be best to do the GITHUB_WORKSPACE stuff as much as possible in here and not in cifuzz.py so that it can be used in other places more easily.

If there's a pythonpath issue that should be resolved by putting cifuzz.py in cifuzz/ (though this might need to be hoisted to that dir as well, instead I'm fine with appending to the pythonpath).

Reply:
This is good advice. I refactored it to use cifuzz as a module.